README.md

```diff
@@ -49,6 +50,26 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### Mac OS
+
+ddev Cypress Setup (Mac)
+
+```bash
+# Install XQuartz
+brew install xquartz --cask
+
+# Run XQuartz
+open -a Xquartz
+```
+checkbox checkedIn the XQuartz preferences, go to the “Security” tab and make sure you’ve got the “Allow connections from network clients” ticked:
+```bash
+# Run the below command
+xhost + 127.0.0.1
+```
+Open `docker-compose.cypress.yaml` and in the *environment* section add 
```

Comment:
Rather than editing the `docker-compose.cypress.yaml`

Option 1:
Add a `docker-compose.cypress_extra.yaml`, which will add to the existing, but not make it so the add-on can't be updated.

Option 2:
Consider adding this into this project. I am not at all sure this won't work for all situations including WSL2.